bin/restore.sh

```diff
@@ -9,7 +9,12 @@ restore_file=${1}
 if /app/bin/django_wait_for_db.sh
 then
 
-    echo "Dropping existing DB."
```

Comment:
Since this is a destructive operation that we don't need to automate, I recommend adding a confirmation before even checking the database, eg.
```
read -p "This will drop the DB. Proceed [y/N]? " -n 1 -r
echo ""
if [[ ! $REPLY =~ ^[Yy]$ ]]
then
    echo "Exiting."
    exit
fi
```